src/sys/epoll.rs

```diff
@@ -1,5 +1,6 @@
 use crate::errno::Errno;
 pub use crate::poll_timeout::PollTimeout as EpollTimeout;
+pub use crate::poll_timeout::PollTimeoutTryFromError;
```

Comment:
```suggestion
pub use crate::poll_timeout::PollTimeoutTryFromError as EpollTimeoutTryFromError;
```

Reply:
I could, but I think this would lead in even more confusion. Consider this page: https://docs.rs/nix/latest/nix/sys/epoll/struct.EpollTimeout.html

It mentions `EpollTimeout` 3 times, it mentions the original name (`PollTimeout`) 68 times.

Reply:
(did it anyway, it's not _that_ important)